src/components/pages/blog-post/blog-post.tsx

```diff
@@ -27,45 +33,148 @@ const PanelContainer = styled.div`
   }
 `;
 
-export const BlogPostPage = ({ markdown, breadcrumb }: BlogPostPageProps) => {
+const contentfulRenderOptions: Options = {
```

Comment:
Note to ourselves, this makes a great separate file to collect all the wonderful mappings here. I'm still amazed how this is very similar to our rehype approach with markdown. Like we picked the perfect solution before even knowing the contentful migration will benefit from it so much. Superb job 🙌

Reply:
I was also surprised how smooth that one went. Was basically plug&play. 😀